src/scripts/CMSSync/index.ts

```diff
@@ -0,0 +1,103 @@
+import fs from 'fs'
+import moment from 'moment'
+
+import api from '../api'
+import settings from './settings'
+import { Ifo, PoolConfig, FarmConfig } from '../../config/constants/types'
+import { SettingsObject } from './types'
+
+const getIfos = (data) => {
+  const ifos: Ifo = data.map((obj) => {
+    const lunchTime = moment(obj.launch_datetime)
+    return {
+      id: obj._id,
+      isActive: obj.is_active,
+      address: obj.address,
+      name: obj.name,
+      subTitle: obj?.sub_title,
+      description: obj?.description,
+      launchDate: lunchTime.format('YYYY-MM-DD'),
+      launchTime: lunchTime.format('h:mm:ss'),
+      saleAmount: obj.sale_amount,
+      raiseAmount: obj.raise_amount,
+      cakeToBurn: obj.burn_amount,
+      projectSiteUrl: obj.project_url,
+      currency: obj.currency,
+      currencyAddress: obj.currency_address,
+      tokenDecimals: obj.decimals,
+      releaseBlockNumber: obj.__v,
+    }
+  })
+  return ifos
+}
+
+const getPools = (data) => {
+  const pools: PoolConfig = data.map((obj) => {
+    return {
+      sousId: obj._id,
+      // image: string,
+      tokenName: obj?.token?.name,
+      stakingTokenName: obj?.quote_token?.name,
+      stakingLimit: obj?.quote_token?.decimals,
+      stakingTokenAddress: obj?.quote_token?.mainnet_address,
+      contractAddress: obj.contract_address,
+      poolCategory: obj.category,
+      projectLink: obj.project_url,
+      tokenPerBlock: obj.token_per_block,
+      sortOrder: obj.ranking,
+      harvest: obj.harvest,
+      isFinished: obj.is_finished,
+      tokenDecimals: obj?.token?.decimals,
+    }
+  })
+  return pools
+}
+
+const getFarms = (data) => {
+  const farms: FarmConfig = data.map((obj) => {
```

Comment:
Please use descriptive names. This should be `farm` instead of `obj`

Reply:
Agreed!